config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get "up" => "rails/health#show", as: :rails_health_check
+
+  # Defines the root path route ("/")
+  # root "posts#index"
+end
```

Comment:
- [Optional] Hi @Zuwairanajma, its like you're not testing the piece of codes you are writing. I think it is very easy to detect any bug at the very beginning. I'm saying this because you haven't add any route. The router is like a gatekeeper for any http requests to test your app. My hypothesis may not be correct since you might have used rails console :smile:

Reply:
> * [Optional] Hi @Zuwairanajma, its like you're not testing the piece of codes you are writing. I think it is very easy to detect any bug at the very beginning. I'm saying this because you haven't add any route. The router is like a gatekeeper for any http requests to test your app. My hypothesis may not be correct since you might have used rails console 😄

@FrederickMih, thanks so much for this observation, I used the console.rails  like you guessed and now I have learned and added the routes, I appreciate your constructive review